osi_hostvehicledata.proto

```diff
@@ -2,31 +2,356 @@ syntax = "proto2";
+        //
+        // Unit: rad/s
+        //
+        optional double angular_speed = 2;
+
+        // Torque of the steering wheel to the hand.
+        // 0=Central (Straight); Left>0; 0>Right.
+        //
+        // Unit: N*m
+        //
+        optional double torque = 3;
+    }
+
+    //
+    // \brief The focus here is on the description of the wheels.
+    //
+    message VehicleWheels
+    {
+        // Contains the description of a wheel.
+        //
+        repeated Wheel wheel = 1;
+
+        //
+        // \brief The focus here is on the description of a wheel.
+        //
+        message Wheel
+        {
+            // The axle which contains this wheel.  A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            //
+            optional uint32 index = 2;
+
+            // Dry friction is a force that opposes the relative lateral motion of two solid surfaces 
+            // in contact. It is subdivided into static friction between non-moving surfaces and kinetic
+            // friction between moving surfaces.
+            // Ued here is the dry friction coefficient of the paired materials (see reference).
+            // Dimensionless.
+            //
+            // Unit: -
+            //
+            // \par References:
+            // - http://adaptivemap.ma.psu.edu/websites/6_friction/dry_friction/dryfriction.html
+            //
+            //optional double kinetic_friction_coefficient = 3;
+
+            // Contact point of the mid of the tire tread with the world.
+            // Absolute coordinates (x, y, z).
+            //
+            // Unit: m
+            //
+            //optional Vector3d contact_point = 4;
+
+            // Contains the rotation rate of the wheel per second in radians.
+            // Based on the processed output of the hall sensor at the wheel.
+            //
+            // Unit: rad/s
+            //
+            optional double rotation_rate = 5;
+
+            // Contains the longitudinal slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            optional double slip = 6;
+        }
+    }
+
+    //
+    // \brief This message contains all the information the vehicle knows about its kinematic states.
+    //
+    message VehiclePositionAndKinematics
+    {
+        //
+        // \brief Current kinematic data based on the output of the Intertial Measurement Unit (IMU).
```

Comment:
This position, and likely also the Georeferenced Information, are unlikely to be the direct output of any IMU or GPS directly, but are the output of the overall position estimation function of the host vehicle, which might combine multiple location sources (IMU, multiple GPS receivers, other sensors, cross-correlation etc.). Raw sensor outputs are travelling via SensorData, or potentially needed further messages. I think this should be made clearer in the descriptions...

Reply:
I agree, that it has to made clear if it is postprocessed/combined data or raw. But where does sensorics start and end? Is a rpm measurement sth for sensorData or is there the line to hostVehicleData passed?
Sth I am interested in and for the next CCB!